docs/user/dashboard/lens.asciidoc

```diff
@@ -54,6 +54,27 @@ For array value fields, the percentage distribution considers each value in the
 
 TIP: *Other* can equal more than 100% by a small amount.
 
+[float]
+[[add-fields-in-lens]]
+===== Add fields
+
+Add and define fields that you want to visualize using the {ref}/modules-scripting-painless.html[Painless scripting language].
+
+. Click *...*, then select *Add field to index pattern*.
++
+[role="screenshot"]
+image:{kib-repo}/discover/images/add-field-to-pattern.png[Dropdown menu located next to index pattern field with item for adding a field to an index pattern, width=50%]
```

Comment:
This image is broken in the preview and it looks like it's a discover picture. Should we have a separate Lens screenshot?